cowprotocol/watch-tower
# feat: unify logging

# Description
Provides unification around logging levels and methodology given the movement to our cluster / ELK logging.

# Changes

- [x] Uses `loglevel`
- [x] Removes sentry / loggly

## How to test
Run and observe logging differences. If desiring granular logs, set environment variable `LOG_LEVEL` to 'DEBUG' or 'TRACE'.

## Related Issues

Fixes #67
